### PR TITLE
Book store initial

### PR DESCRIPTION
**Pull request by mahdimahdawi**
**This PR Contain:**
- Initialised React app to repo.
- Created a directory with the name of the components.
-  Added React Route and set two `Route`s and `Link`s for the app's navigation.
- Created Books component and set the default view.
- Created a component to display the list of books with the Remove button.
- Created a component called Book to display a single book that receives a title and an author as prop.
- Created a separate component for the form containing input fields for title and author and also submit button.
- Created Categories component that displays a button "Check status" only.

### CODE REVIEW

**Review by phelian23 — COMMENTED** [review]
Hi @mahdimahdawi,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no linter set up. Kindly set up the appropriate linter for this project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Comment by mahdimahdawi** [comment]
- Added `.github` directory.

File: README.md [hunk 1]
```diff
@@ -1,75 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Getting Started with Create React App
```
Comment (Whoistolu) [inline README.md]: - [x] [REQUIRED] Great project implementation, however, you need to update the README with the correct details for the project for more professionalism, currently, we have the `create-react-app` default README. Kindly fix this to get this project approved.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @mahdimahdawi 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mahdimahdawi** [comment]
**REQUIRED CHANGES IMPLEMENTED**
- Updated `README.md` from `create-react-app` default to an appropriate date.

**Review by m05e5 — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @mahdimahdawi  🔥 

Good job on implementing the required cnahges. Well Done.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

goodjob


Now you can merge this branch 🚀 

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

_m05e5_

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**